apis/v1beta1/httproute_types.go

```diff
@@ -164,6 +164,7 @@ type HTTPRouteRule struct {
 	// * Characters in a matching "Prefix" path match
 	// * Header matches.
 	// * Query param matches.
+	// * Method matches.
```

Comment:
curious why its after `Query param` ? is it to maintain backwards compatibility ?

Reply:
+1. I would put it before "Headers" personally

Reply:
(also seems like the language before this list needs to be amended, currently it reads `precedence must be given to the match with the largest number of:`, and adding method matching in here doesn't really fit)

Reply:
@gauravkghildiyal at today's meeting I think we had consensus that this should fit right above header matches. Can you update that and the corresponding conformance test?

Reply:
Thanks for the info. Updated!